### PR TITLE
Add sinch SMS connector to main branch

### PR DESCRIPTION
**Pull request by sachinira**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

Ballerina SL Beta 3
Ubuntu 20.04

### CODE REVIEW

File: openapi/sinch.sms/Dependencies.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "1.1.0-beta.2"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "1.1.0-beta.2"
```
Comment (Chuhaa) [inline openapi/sinch.sms/Dependencies.toml:9]: Can you update the beta3 versions?

File: openapi/sinch.sms/Module.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+## Overview
+This is a generated connector for [Sinch SMS API v2.0](https://www.sinch.com/) OpenAPI specification. 
+
+Sinch SMS API is the one of the easiest APIs we offer and enables you to add fast and reliable global SMS to your 
+applications. It allows for sending single messages, scheduled batch sending, message templates and more.
```
Comment (Chuhaa) [inline openapi/sinch.sms/Module.md:5]: We can merge this line to previous line.

File: openapi/sinch.sms/Package.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Connects to [Sinch SMS API v1](https://www.sinch.com/) from Ballerina
+
+## Package overview
+The `ballerinax/sinch.verification` is a [Ballerina](https://ballerina.io/) connector connecting to Sinch SMS API.
+
+### Compatibility
+|                       | Version                       |
+|-----------------------|-------------------------------|
+| Ballerina Language    | Ballerina Swan Lake Beta3     | 
+| Sinch SMS API         | v1                            |
```
Comment (Chuhaa) [inline openapi/sinch.sms/Package.md:10]: Different version is in Module.md. Please update the correct version.

File: openapi/sinch.sms/openapi.yaml [hunk 4]
```diff
@@ -0,0 +1,1657 @@
...
+  description: 
+    This is a generated connector for [Sinch SMS API v1](https://www.sinch.com/) OpenAPI specification. 
+
+    Sinch SMS API is the one of the easiest APIs we offer and enables you to add fast and reliable global SMS to your applications. It allows for sending single messages, scheduled batch sending, message templates and "Only list messages sent from this sender number. Multiple originating numbers can be comma separated.
+  x-ballerina-init-description:
+    The connector initialization requires setting the API credentials.
+
+    Create a [Sinch account](https://www.sinch.com/) and obtain tokens by following [this guide](https://developers.sinch.com/docs/sms/api-reference/#authentication).
+  x-ballerina-display:
+    label: sinch
```
Comment (Chuhaa) [inline openapi/sinch.sms/openapi.yaml:14]: Don't we need to give it as Sinch SMS? Image name is it as `sinch.sms`.

File: openapi/sinch.sms/Module.md (hunk 2 above)
Comment (sachinira) [reply openapi/sinch.sms/Module.md]: Merged. thanks

File: openapi/sinch.sms/Dependencies.toml (hunk 1 above)
Comment (sachinira) [reply openapi/sinch.sms/Dependencies.toml]: Updated

File: openapi/sinch.sms/openapi.yaml (hunk 4 above)
Comment (LakshanSS) [reply openapi/sinch.sms/openapi.yaml]: Yes, `Sinch SMS` will be the correct name

**Review by Chuhaa — APPROVED** [review]

File: openapi/sinch.sms/.gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+target
+tests
```
Comment (LakshanSS) [inline openapi/sinch.sms/.gitignore:2]: We decided to remove `tests` here right?
@sachinira @Chuhaa

**Review by RolandHewage — APPROVED** [review]

**Review by LakshanSS — APPROVED** [review]

**Review by RolandHewage — APPROVED** [review]

**Review by RolandHewage — APPROVED** [review]

**Pull request merged**